ovh/client.py

```diff
@@ -335,15 +335,21 @@ def get(self, _target, _need_auth=True, **kwargs):
         :param string _need_auth: If True, send authentication headers. This is
             the default
         """
+
+        batch = None
+
         if kwargs:
             kwargs = self._canonicalize_kwargs(kwargs)
+            batch = kwargs.pop('batch', None)
```

Comment:
I would change it to ```_batch``` like we did with ```_need_auth``` to prevent conflicts with actual body parameters

Reply:
Done